spinta/manifests/tabular/helpers.py

```diff
@@ -859,6 +859,46 @@ def leave(self) -> None:
         pass
 
 
+class UniqueReader(TabularReader):
+    type: str = 'unique'
+
+    def read(self, row: ManifestRow) -> None:
+        reader = self.state.stack[-1]
+
+        if not isinstance(reader, (
+            DatasetReader,
+            ResourceReader,
+            BaseReader,
+            ModelReader,
+            PropertyReader,
+        )):
+            return
+
+        if self.type not in reader.data:
+            reader.data['unique'] = {}
+
+        unique = reader.data['unique']
+
+        self.name = '_'.join(row[REF].split(', '))
```

Comment:
Split just by ',' and then strip spaces. Because `ref` could be given as `name,country` or `  name,     country  `.